modules/images/src/lib/parsers/svg-utils.ts

```diff
@@ -12,11 +12,11 @@ export function getBlobOrSVGDataUrl(arrayBuffer, url) {
   if (isSVG(url)) {
     // Prepare a properly tagged data URL, and load using normal mechanism
     const textDecoder = new TextDecoder();
-    const xmlText = textDecoder.decode(arrayBuffer);
+    let xmlText = textDecoder.decode(arrayBuffer);
     // TODO Escape in browser to support e.g. Chinese characters
-    // if (typeof unescape === 'function' && typeof encodeURLComponent === 'function') {
-    //   xmlText = unescape(encodeURLComponent(xmlText));
-    // }
+    if (typeof unescape === 'function' && typeof encodeURIComponent === 'function') {
+      xmlText = unescape(encodeURIComponent(xmlText));
+    }
```

Comment:
I wonder why this was commented out. It may have caused some issues. Maybe wrap this in a `try catch` as well?
```suggestion
   try {
     if (typeof unescape === 'function' && typeof encodeURIComponent === 'function') {
      xmlText = unescape(encodeURIComponent(xmlText));
    }
   catch () {
   }
```